docs/PokemonCreation.md

```diff
@@ -0,0 +1,70 @@
+## Pokemon Creation
+
+The two main files that handle Pokemon creation are [`Equations.ts`](../src/components/Equations.ts) and [`Utilities.ts`](../src/components/Utilities.ts).
+`Equations.ts` includes functions for choosing random wild Pokemon and for creating the actual Pokemon object.
+`Utilities.ts` facilitates the creating of a wild Pokemon by preparing parameters then calling `Equations.ts` creation of the Pokemon object.
+
+A newly created Pokemon's member types can be found in [`Battles`.ts](../src/components/Battles.ts) under the IPokemon interface.
```

Comment:
Throughout this file, go through the function and interface names and backtick-highlight them. A lot are missing highlighting.